pricing-page-exercise/mesut/index.html

```diff
@@ -1,12 +1,117 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML 5 Boilerplate</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML 5 Boilerplate</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/5e08b15075.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="company-name"><a href="#" class="company-link link">Company name</a></div>
+    <div class="top-right-links">
+      <ul class="top-right-buttons">
+        <li class="top-right-list"><a href="#" class="top-right-buttons link features-button">Features</a></li>
```

Comment:
I'd use `navbar-list-item` class for `li` element, and `navbar-buttons link`.
I see that you used one more class for each button like `features-button` in this case to add styling `text-decoration: underline;` to all the link elements in navbar. I suggest using a `navbar-buttons` or `link` class for that instead to minimize code duplication.

Reply:
,